### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by francoisferrand**

### CODE REVIEW

File: .github/workflows/actions.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        # Fix libpcre2-posix2 package version, use the 
+        sudo apt-get -y install --reinstall --allow-downgrades libpcre2-posix2/$(lsb_release -cs)
+
+        # Also need oras, skopeo ; but they are already installed
+
+        bash <(curl -s https://raw.githubusercontent.com/pgrange/bash_unit/master/install.sh)
+
+        # Start cri-o service
+        sudo systemctl start crio
```
Comment (lavigne958) [inline .github/workflows/actions.yaml:76]: This is tipicaly the kind of things you want to setup in a docker image then use that image to run.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.RELEASE_VERSION }}
+          release_name: Release ${{ env.RELEASE_VERSION }}
```
Comment (lavigne958) [inline .github/workflows/release.yml]: we have a sample file already used by multiple repositories to do a release, it adds a new action you can trigger manually using github ui. may be we should take the same sample file here so we keep it consistent with every repo.

BTW: you miss the trailing new line.

File: .github/workflows/actions.yaml (hunk 1 above)
Comment (francoisferrand) [reply .github/workflows/actions.yaml:76]: yes, but it uses CRI-O and systemd, which do not work in a docker image....

File: .github/workflows/release.yml (hunk 2 above)
Comment (francoisferrand) [reply .github/workflows/release.yml]: i don't want to do it manually : pushing the (git) tag is already done  manually, so the rest should be done automatically.

(though i agree some uniformity may be good, but at the moment we don't have a "final", well-integrated solution... we plan to discuss this with Thomas later in the week, but I'll try to push the boundaries of the known world for now)

Comment (francoisferrand) [reply .github/workflows/release.yml]: (fixed the trailing new line)

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: registry.scality.com
+          username: ${{ secrets.REGISTRY_LOGIN }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```
Comment (lavigne958) [inline .github/workflows/release.yml]: Please use the appropriate variable name: `GITHUB_REF_NAME`
See [here](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)

**Review by lavigne958 — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (lavigne958) [reply .github/workflows/release.yml]: :stuck_out_tongue:

Comment (francoisferrand) [reply .github/workflows/release.yml]: much simpler indeed!
and some more doc [here](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)

**Review by lavigne958 — APPROVED** [review]

**Review by lavigne958 — APPROVED** [review]

**Pull request merged**